posthog/migrations/0143_user_uuid.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.0.11 on 2021-04-02 23:11
+
+from django.db import migrations, models
+
+import posthog.models.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0142_fix_team_data_attributes_default"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="user", name="last_name",),
```

Comment:
So we don't actually use this field, BUT Google auth does and some of our Google auth-using users have it, even though they can't see it. I think we should put a migration before this `RemoveField` to rename users to
```Python
for user in User.objects.exclude(last_name=""):
    user.first_name = f'{user.first_name} {user.last_name}'
    user.save()
```
And we should verify that user registration with G auth works after this field is gone

Reply:
Makes sense, will keep this for a separate improvement.